sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -935,6 +935,7 @@ public void forceInsert(BasicDocumentRevision rev,
                 }
             }
             if (ok) {
+                logger.finer("Inserted revision: "+rev);
```

Comment:
Is there a logging method which takes a formatting string and the object so we can avoid the rev.toString and concat calls for most runs through this loop?